lib/DidIo.js

```diff
@@ -34,9 +34,10 @@ class DidIo {
     this.methods[methodName] = driver;
   }
 
-  async get({did} = {}) {
+  async get({did, forceConstruct} = {}) {
```

Comment:
This should really just be additional options that get passed through to the specific driver -- then we don't have to make a `forceConstruct` decision at this layer.

Reply:
+1 for this decision not being made at this layer. The idea of `forceConstruct` is specific to a DID Method Driver's implementation. This is not a general feature of all drivers.